test/internal/modules/charge-information/pre-handlers.js

```diff
@@ -531,6 +592,8 @@ experiment('internal/modules/charge-information/pre-handlers', () => {
         const [,, chargeVersion] = request.setDraftChargeInformation.lastCall.args;
         expect(chargeVersion.status).to.equal('review');
         expect(chargeVersion.invoiceAccount.invoiceAccountAddress).to.equal('test-invoice-account-address-id');
+        expect(chargeVersion.chargeElements[0].id.length === 36).to.be.true();
```

Comment:
Would it not be better to expect the id to be equal to a string and not just the length of the string?

Reply:
changed this to use Joi as Tim suggested thanks